src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -97,12 +98,16 @@ export const TradeInput = ({ isCompact, tradeInputRef, onChangeTab }: TradeInput
     useAccountIds()
   const buyAssetSearch = useModal('buyTradeAssetSearch')
 
+  const isThorchainSwapperEnabled = useFeatureFlag('ThorchainSwapperAck')
```

Comment:
```suggestion
  const isThorchainSwapperVolatilityAckEnabled = useFeatureFlag('ThorchainSwapperVolatilityAck')
```

Reply:
and same vernacular all around too

Reply:
Done in [2446009](https://github.com/shapeshift/web/pull/8718/commits/2446009104efd392993d9893fa0161f4e9ed33c1)